Stripe/STPAPIClient.m

```diff
@@ -260,9 +260,42 @@ - (void)createTokenWithPersonalIDNumber:(NSString *)pii completion:(__nullable S
 
 @implementation STPAPIClient (Upload)
 
+- (NSData *)dataForUploadedImage:(UIImage *)image
```

Comment:
@jack-stripe Do you think this is a reasonable auto image scaling algorithm? (just keep redoing the image at 0.8x the previous scale until its small enough). Also not sure how to test this, I guess I could just check in some really large photos into our repo?